### ISSUE TITLE
64-bit version?

### ISSUE THREAD

**Issue by imsdala** [DonnchaC/coldboot-attacks#1]
I have tried creating a 64-bit version using Ubuntu 16.04 with:
make -f Makefile.64
and then sudo dd if=scraper.bin of=/dev/sdb to no avail.

Have tried dumping the memory but I only get some text:
Bootstrap loaded.. trying pakcet mode .. starting
USB Memory scraper, written by Bill Paul..

Then it just restarts. Tried it on multiple computers. Any ideas?

**Comment by Flova** [comment]
I have the same problem with the 64bit Version. 32 bit would be also fine for me but it crashes while building saying it doesn't support x86.

**Comment by n-92** [comment]
yep.. same issue occuring.. someone suggested setting -m32 flag instead of the march=i386. But that also doesn't work because the instruction sets for the assembly file throw error.
